### ISSUE TITLE
configure AWS resources - Cognito user pool

### ISSUE THREAD

**Issue by fyliu** [hackforla/peopledepot#326]
### Dependency

 - #325 may contain instructions

### Overview

As part of our AWS configuration, we need to work with #ops to configure a Cognito user pool.

### Action Items

- [ ] follow documentation (if any) to create configs to provision a Cognito user pool
- [ ] meet with #ops to verify the setup (through the PR process)
- [ ] link to the configuration from here and from our documentation

### After Merge

- [ ] check off this issue in #327 and close it if done

### Resources/Instructions

- requirements:
   - email required
   - allow self-signup
   - send email and verify email address
   - no other required attributes
   - no MFA(for now)
- look at the existing VRMS testing user pool if we're missing other configs

**Comment by fyliu** [comment]
We may need an issue to define what options we want for the user pool. I think in my testing one, the only thing required is email address.

**Comment by gmgonzal** [comment]
Update ETA: 3/20

**Comment by shmonks** [comment]
@gmgonzal If there's a question you'd like to ask, could you add it in a comment below?

**Comment by gmgonzal** [comment]
- [ ] I synced with devops on this one. Devops is under the impression that PeopleDepot is utilizing the VRMS user pool. If this the case, then the question here is would we like to keep using the VRMS user pool or create a new one?

**Comment by fyliu** [comment]
Yes, we agreed to keep using the VRMS user pool for development. So we don't really need this issue anymore.
